python/py-vitables/Portfile

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           select 1.0
+
+name                py-vitables
+version             3.0.3
+categories          python science
+platforms           darwin
+maintainers         nomaintainer
+license             GPL-3
+
+description         ViTables is a graphical tool for PyTables.
+
+long_description    ViTables is a graphical browser for HDF5 and PyTables files. It allows you to \
+                    open HDF5 files, browse their contents and display datasets.
+
+homepage            https://github.com/uvemas/ViTables
+master_sites        pypi:v/vitables
+
+checksums           sha256  24570963c1a2fc8df3ecb39a736256a5384aff114b56540c6bdad7c4197881a0 \
+                    rmd160  f3f66b119afa25b018830f1d10d8c176e44164e3 \
+                    size    871792
+
+python.default_version  311
+python.versions     38 39 310 311 312
```

Comment:
is this a program that happens to be written in Python or a module that's intended to be imported? To me it seems the former and in that case what's the reason/point to provide it for different Python versions?
Why not call it "vitables" and inly install it with Python 3.12 (the current default Python version). That would go remove the need for all this select files as well. Let's first figure that out and then we can look at the detail in the Portfile.

Reply:
You're right. I've changed the PR and removed the select files.